src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ProcessorTestSuite.java

```diff
@@ -124,6 +126,8 @@ public void setUp() throws Exception {
 
         blobManager = new BlobManager(cloudBlobClient, blobManagementProperties);
 
+        fileNamesExtractor = new FileNamesExtractor();
```

Comment:
can it not be autowired?

Reply:
FileNamesExtractor depends on _scheduling.task.scan.enabled_ property (along with BlobProcessorTask) which is false in integration tests.